src/arithmetic/arithmetic_expression.c

```diff
@@ -365,13 +555,8 @@ void AR_EXP_Free(AR_ExpNode *root) {
         if (root->op.type == AR_OP_AGGREGATE) {
             AggCtx_Free(root->op.agg_func);
         }
-    } else {
-        if (root->operand.type == AR_EXP_CONSTANT) {
-            SIValue_Free(&root->operand.constant);
-        } else {
-            if (root->operand.variadic.entity_alias) rm_free(root->operand.variadic.entity_alias);
-            if (root->operand.variadic.entity_prop) rm_free(root->operand.variadic.entity_prop);
-        }
+    } else if (root->operand.type == AR_EXP_CONSTANT) {
+        SIValue_Free(&root->operand.constant);
```

Comment:
Good practice, but are we really expecting the SIValue to free anything?

Reply:
I doubt it! String manipulation functions like `toLower(a.name)` allocate SIValues that must be freed, but off the top of my head, I can't think of a way for those SIValues to get here.